DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseOptions.cshtml

```diff
@@ -0,0 +1,33 @@
+﻿@inject IConfiguration Configuration;
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@using Microsoft.Extensions.Configuration
+@model EditCourseOptionsViewModel
+
+@{
+  ViewData["Title"] = "Edit course options";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
```

Comment:
This should use _AutoNavMenuItems for the nav and should use `ApplicationType.TrackingSystem` to set Application, HeaderPath and HeaderPathName.